mlir/lib/Dialect/QTensor/IR/Operations/ExtractOp.cpp

```
@@ -9,10 +9,15 @@
  */
 
 #include "mlir/Dialect/QTensor/IR/QTensorOps.h"
+#include "mlir/Dialect/QTensor/IR/QTensorUtils.h"
```

Cppcheck (2.20.0)

[information] Include file

(missingInclude)